### DISCUSSION TITLE
Vehicle icon is not removed in the web visualizer after the vehicle leave the scene

### DISCUSSION THREAD

**Discussion by DFCui** [eclipse/mosaic#150] [Q&A] [answered]
Hi all,
I run the Tiergarten scenario with a long simulation time and found the vehicle icons are still there after the vehicles leave the scene. Here are screenshots. 
![image](https://user-images.githubusercontent.com/24232678/135672283-b5da5aa2-3051-49ca-8c89-05c91a277d0e.png)
![image](https://user-images.githubusercontent.com/24232678/135672355-9fd3e675-b297-48be-9fce-09c9bd4efb27.png)
From the image of SUMO, you can see that there are no vehicles in the scenario. However, from the image of web visualizer, the vehicle icons are still there. 

Moreover, I found vehicle icons in the web visualizer is updated every around 1s. The update interval does not change when I set a small time step for SUMO., for example, 0.1, 0.2. 

Is  there  anyone having the same issues?  Could you please providing some suggestions  for  solving them?

Thank you very  much.

**Comment by kschrab** [comment, marked as answer]
> From the image of SUMO, you can see that there are no vehicles in the scenario. However, from the image of web visualizer, the vehicle icons are still there.

This is indeed an issue. We will investigate how this can be fixed.

> Moreover, I found vehicle icons in the web visualizer is updated every around 1s. The update interval does not change when I set a small time step for SUMO., for example, 0.1, 0.2.

It's currently not possible to go below 1 second for the update interval of the web visualizer. In the `OutputAmbassadorLoader` the update interval is currently returned as an integer. If you want to change this, feel free to make a pull request.

**Reply by DFCui** [reply]
Thank you very much for your quick answers.

**Reply by kschrab** [reply]
> > From the image of SUMO, you can see that there are no vehicles in the scenario. However, from the image of web visualizer, the vehicle icons are still there.
> 
> This is indeed an issue. We will investigate how this can be fixed.

This will be fixed with #151

**Reply by DFCui** [reply]
Thank you very much for your quick fixing the issue.
